Source/WebCore/page/FrameView.cpp

```diff
@@ -2999,6 +2999,13 @@ void FrameView::addedOrRemovedScrollbar()
     }
 
     updateTiledBackingAdaptiveSizing();
+
+    if (UNLIKELY(InspectorInstrumentationPublic::hasFrontends())) {
+        auto* document = frame().document();
+        auto* documentElement = document ? document->documentElement() : nullptr;
+        if (documentElement)
+            documentElement->notifyInspectorOfRendererPropertyChange();
+    }
```

Comment:
Hmm this is not really the normal way we do `InspectorInstrumentation` things.  I only did this with `Node::notifyInspectorOfRendererChange` because we couldn't put `InspectorInstrumentation` calls in a header file (which is where the definition of `Node::setRenderer` was) and to try to make `Node::setRenderer` a bit faster by avoiding extra function calls.  It's possible that the latter part of that is not really a concern (anymore), and I instead could've just called `notifyInspectorOfRendererChange` directly (i.e. without being guarded by `if (UNLIKELY(InspectorInstrumentationPublic::hasFrontends()))`), but at the time I was told it's something to avoid.

If that's not something we have to worry about here, I think it'd be better to just `InspectorInstrumentation::addedOrRemovedScrollbar(*this)` and then let `InspectorCSSAgent::addedOrRemovedScrollbar` handle getting the `documentElement` from the `FrameView`.  That way, it keeps (as much as possible of) the inspector-specific logic inside inspector objects.